### PR TITLE
Added search bars to the Dust and Canister tab and added the empty canis...

### PR DESCRIPTION
**Pull request by iTitus**
...ters to the main tab.

Should make @BigXplosion happy ;)

### CODE REVIEW

File: src/main/resources/assets/bcadditions/lang/en_US.lang [hunk 1]
```diff
@@ -133,7 +133,7 @@ priority.high=High
 priority.veryHigh=Very High
 
 itemGroup.BuildcraftAdditions=Buildcraft Additions
-itemGroup.bcaCanisters=Fluid Canisters
+itemGroup.bcaCanisters=Canisters
```
Comment (AEnterprise) [inline src/main/resources/assets/bcadditions/lang/en_US.lang]: why this change?

Comment (BigXplosion) [inline src/main/resources/assets/bcadditions/lang/en_US.lang]: I agree doesn't make sense...

Comment (iTitus) [inline src/main/resources/assets/bcadditions/lang/en_US.lang]: Otherwise it wont fit with the search bar...

**Comment by iTitus** [comment]
Closing in favor of #221
